tests/Testing/Temporary/LastTest.log

```diff
@@ -1,3 +1,3 @@
-Start testing: Sep 11 08:44 PDT
```

Comment:
why is this file in Git and why do you update it? `Temporary` kind suggests that it should not be under version control at all...